src/sbt-test/checkstyle/checkstyle-auto/build.sbt

```diff
@@ -1,11 +1,9 @@
-version := "0.1"
+version := "2.1.1-SNAPSHOT"
```

Comment:
Are they recommending changing the version in all the scripted tests to match the version of the plugin itself?